frontend/src/App.tsx

```diff
@@ -1270,6 +1273,32 @@ export class App extends PureComponent<Props, State> {
     this.openDialog(newDialog)
   }
 
+  printCallback = (): void => {
+    if (isInChildFrame()) {
```

Comment:
Can you describe this logic in a comment?

Reply:
Sure, it's updated 👍